examples/decoding/plot_mixed_gambles_space_net.py

```diff
@@ -0,0 +1,45 @@
+"""
+SpaceNet on Jimura et al "mixed gambles" dataset.
+==================================================
+
+The segmenting power of SpaceNet is quite visible here.
```

Comment:
On Tue, Jul 28, 2015 at 5:22 PM, Gael Varoquaux <notifications@github.com>
wrote:

> In examples/decoding/plot_mixed_gambles_space_net.py
> <https://github.com/nilearn/nilearn/pull/657#discussion_r35660156>:
>
> > @@ -0,0 +1,45 @@
> > +"""
> > +SpaceNet on Jimura et al "mixed gambles" dataset.
> > +==================================================
> > +
> > +The segmenting power of SpaceNet is quite visible here.
>
> Please explain here that you are using the regressor object given that the
> task is to predict a continuous variable, the gain of the gamble.
>
Yeah, definitely.

> Also, maybe we should leave out some data and show a prediction power (if
> it somewhat works).
>
hmm this looks like a big change. I propose this be done separately,
otherwise we won't simply ever be able to merge this PR. I propose we don't
produce a single line of additional code (except fixing bugs, typos, etc.)
related to SpaceNet until this PR is merged :)

> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nilearn/nilearn/pull/657/files#r35660156>.
>



-- 
DED